msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md

```diff
@@ -40,31 +40,30 @@ To use these permissions, you must add a [webApplicationInfo](../../resources/sc
 > [!div class="checklist"]
 > [!div class="checklist"]
 >
-> * **id**  — your Azure AD app id.
+> * **id**  — your Azure AD app ID.
 > * **resource** — the resource URL for the app.
 >
-
 >[!NOTE]
 >
-> * Your bot requires _application_ not _user delegated_ permissions because the installation is not for yourself but for others.
+> * Your bot requires application and not user delegated permissions because the installation is for others.
```

Comment:
Who are these **others** in the note?
```suggestion
> * Your bot requires *application* and not *user delegated* permissions because the installation is for others.
```